drivers/serial/Kconfig.lpc11u6x

```diff
@@ -0,0 +1,41 @@
+# LPC11U6X UART configuration
+
+# Copyright (c) 2020 Seagate Technology
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig UART_LPC11U6X
+	bool "LPC11U6X serial driver"
+	select SERIAL_HAS_DRIVER
+	select SERIAL_SUPPORT_INTERRUPT
+	depends on SOC_SERIES_LPC11U6X
+	help
+	  Enable UART driver for LPC11U6X series
+
+if UART_LPC11U6X
+
+config UART_LPC11U6X_PORT0
+	bool "Enable LPC11U6X UART0"
+	help
+	  Enable UART port 0
+
+config UART_LPC11U6X_PORT1
+	bool "Enable LPC11U6X UART1"
+	help
+	  Enable UART port 1
+
+config UART_LPC11U6X_PORT2
+	bool "Enable LPC11U6X UART2"
+	help
+	  Enable UART port 2
+
+config UART_LPC11U6X_PORT3
+	bool "Enable LPC11U6X UART3"
+	help
+	  Enable UART port 3
+
+config UART_LPC11U6X_PORT4
+	bool "Enable LPC11U6X UART4"
+	help
+	  Enable UART port 4
```

Comment:
Could these be obtained from devicetree instead of Kconfig?

Reply:
Indeed it can, I removed Kconfig option in favor of device tree macro based on the node's status.